examples/teleop-react/src/client.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  createRobotClient,
+  StreamClient,
+  type RobotClient,
+} from '@viamrobotics/sdk';
+
+export interface RobotCredentials {
+  hostname: string;
+  secret: string;
+}
+
+/**
+ * Given a set of credentials, get a robot client.
+ *
+ * @param credentials Robot URL and location secret
+ * @returns A connected client
+ */
+export const getRobotClient = async (
+  credentials: RobotCredentials
+): Promise<RobotClient> => {
+  const { hostname, secret } = credentials;
+
+  return createRobotClient({
+    authEntity: hostname,
+    host: hostname,
+    credential: {
+      type: 'robot-location-secret',
+      payload: secret,
+    },
+    signalingAddress: 'https://app.viam.com:443',
+    iceServers: [],
```

Comment:
we might want the default iceServers to include a free stun server:

```suggestion
    iceServers: [{ urls:"stun:global.stun.twilio.com:3478" }],
```

Have you been able to connect to a robot from another network without this config?